### PR TITLE
[FR] Convert `chat` > `tchat` across the wiki

### PR DESCRIPTION
**Pull request by DoryanR**
Conversion of the term `chat` to `tchat` throughout the wiki for French articles.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Replay/fr.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Pour relancer un replay depuis le début, restez appuyé sur un des boutons par
 L'écran devrait s'assombrir et un son sera joué lorsque le replay sera relancé.
 Relâcher trop tôt ne permettra pas au relancement de s'effectuer.
 
-Il est toujours possible d'accéder à la [console de chat](/wiki/Client/Interface/Chat_console) (`F8`)/[console de chat étendue](/wiki/Client/Interface/Chat_console#la-console-de-chat-étendue) (`F9`) .
+Il est toujours possible d'accéder à la [console de tchat](/wiki/Client/Interface/Chat_console) (`F8`)/[console de tchat étendue](/wiki/Client/Interface/Chat_console#la-console-de-tchat-étendue) (`F9`) .
```
Comment (Walavouchey) [inline wiki/Gameplay/Replay/fr.md:21]: you've changed this link but haven't changed the heading in the chat console article

File: wiki/History_of_osu!/2015/fr.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ Malgré cela, la nouvelle équipe était entièrement publique et pouvait être
 
 ## Août
 
-Les Chat Moderators ont été officiellement fusionnés dans la Global Moderation Team (GMT) le 20 août 2015 [[24,25]][r].
+Les tchat Moderators ont été officiellement fusionnés dans la Global Moderation Team (GMT) le 20 août 2015 [[24,25]][r].
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2015/fr.md:63]: casing?

File: wiki/Registration/fr.md [hunk 3]
```diff
@@ -19,4 +19,4 @@ N'oubliez pas de lire le [règlement](/wiki/Rules) très soigneusement !
 
 Une fois que cela est fait, vous êtes maintenant prêt à commencer votre aventure rythmique ! Vous pouvez [ajouter des beatmaps](/wiki/Client/Installation#ajouter-des-beatmaps) pour jouer, [créer un skin](/wiki/Skinning), ou [créer une beatmap](/wiki/Beatmapping). Vous pouvez également passer et dire "Salut !" dans le [sous-forum Introductions](https://osu.ppy.sh/community/forums/8).
 
-Si vous avez besoin d’une aide supplémentaire dans le jeu, vous pouvez demander dans le canal [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC) `#help` en ouvrant la console de chat (pressez `F8` ou `F9` en jeu) puis tapez `/join help`. Vous pourrez également poser votre question dans le [sous-forum Help](https://osu.ppy.sh/community/forums/5).
+Si vous avez besoin d’une aide supplémentaire dans le jeu, vous pouvez demander dans le canal [Internet Relay tchat](/wiki/Community/Internet_Relay_Chat) (IRC) `#help` en ouvrant la console de tchat (pressez `F8` ou `F9` en jeu) puis tapez `/join help`. Vous pourrez également poser votre question dans le [sous-forum Help](https://osu.ppy.sh/community/forums/5).
```
Comment (Walavouchey) [inline wiki/Registration/fr.md:22]:
```suggestion
Si vous avez besoin d’une aide supplémentaire dans le jeu, vous pouvez demander dans le canal [Internet Relay Chat](/wiki/Community/Internet_Relay_Chat) (IRC) `#help` en ouvrant la console de tchat (pressez `F8` ou `F9` en jeu) puis tapez `/join help`. Vous pourrez également poser votre question dans le [sous-forum Help](https://osu.ppy.sh/community/forums/5).
```

File: wiki/History_of_osu!/2008/fr.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ Pour contrebalancer les nouveaux mods de février, les mods [Relax](/wiki/Game_m
...
+Sans blague, Bancho a été lancé en avril, et avec lui les panneaux d'utilisateurs du [tchat Étendu](/wiki/Client/Interface/Chat_console#la-console-de-tchat-étendue). Les utilisateurs pouvaient maintenant voir ce que les autres utilisateurs faisaient dans leurs panneaux d'utilisateur, et pouvaient également interagir avec eux en cliquant pour les observer. Le système d'envoi de rapports d'erreurs automatique a été ajouté, la compatibilité xfire a été implémentée, et le menu principal et l'interface de l'éditeur ont reçu un relooking (avec les nouvelles icônes par [LuigiHann](https://osu.ppy.sh/users/1079)). L'interface de la console de tchat a vu l'ajout du bouton `Afficher le tchat`, l'onglet `tab completion`, et les alertes de mentions de pseudos. L'écran des scores montrait maintenant le nombre de points requis pour atteindre le prochain rang.
```
Comment (Walavouchey) [inline wiki/History_of_osu!/2008/fr.md:25]: casing? x2

also `Show Chat` isn't even translated in-game

File: wiki/People/The_Team/Language_Surveillance_Division/fr.md [hunk 5]
```diff
@@ -21,7 +21,7 @@ La LSD a également servi de groupe intermédiaire pour les membres prometteurs
 
 ## Restructuration
 
-![](img/chat-moderator-profile.png "Page de profil d'un chat moderator en 2013")
+![](img/chat-moderator-profile.png "Page de profil d'un tchat moderator en 2013")
 
 Après une [restructuration du personnel](https://osu.ppy.sh/community/forums/topics/123510) en 2013, la Language Surveillance Division a été rebaptisée **Chat Moderators** et a reçu sa propre liste de membres.
```
Comment (Walavouchey) [inline wiki/People/The_Team/Language_Surveillance_Division/fr.md:26]: "Chat Moderators" was changed to "tchat Moderators" elsewhere. if you automated this pr, it somehow didn't catch this one

File: wiki/History_of_osu!/2008/fr.md (hunk 4 above)
Comment (DoryanR) [inline wiki/History_of_osu!/2008/fr.md:25]:
```suggestion
Sans blague, Bancho a été lancé en avril, et avec lui les panneaux d'utilisateurs du [Tchat étendu](/wiki/Client/Interface/Chat_console#la-console-de-tchat-étendue). Les utilisateurs pouvaient maintenant voir ce que les autres utilisateurs faisaient dans leurs panneaux d'utilisateur, et pouvaient également interagir avec eux en cliquant pour les observer. Le système d'envoi de rapports d'erreurs automatique a été ajouté, la compatibilité xfire a été implémentée, et le menu principal et l'interface de l'éditeur ont reçu un relooking (avec les nouvelles icônes par [LuigiHann](https://osu.ppy.sh/users/1079)). L'interface de la console de tchat a vu l'ajout du bouton `Afficher le tchat`, l'onglet `tab completion`, et les alertes de mentions de pseudos. L'écran des scores montrait maintenant le nombre de points requis pour atteindre le prochain rang.
```

File: wiki/History_of_osu!/2015/fr.md (hunk 2 above)
Comment (DoryanR) [inline wiki/History_of_osu!/2015/fr.md:63]:
```suggestion
Les Chat Moderators ont été officiellement fusionnés dans la Global Moderation Team (GMT) le 20 août 2015 [[24,25]][r].
```

File: wiki/People/The_Team/Language_Surveillance_Division/fr.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ La LSD a également servi de groupe intermédiaire pour les membres prometteurs
 
 ## Restructuration
 
-![](img/chat-moderator-profile.png "Page de profil d'un chat moderator en 2013")
+![](img/chat-moderator-profile.png "Page de profil d'un tchat moderator en 2013")
```
Comment (DoryanR) [inline wiki/People/The_Team/Language_Surveillance_Division/fr.md:24]:
```suggestion
![](img/chat-moderator-profile.png "Page de profil d'un Chat moderator en 2013")
```

File: wiki/People/The_Team/Language_Surveillance_Division/fr.md [hunk 7]
```diff
@@ -21,7 +22,7 @@ La LSD a également servi de groupe intermédiaire pour les membres prometteurs
 
 ## Restructuration
 
-![](img/chat-moderator-profile.png "Page de profil d'un chat moderator en 2013")
+![](img/chat-moderator-profile.png "Page de profil d'un Chat moderator en 2013")
```
Comment (DoryanR) [inline wiki/People/The_Team/Language_Surveillance_Division/fr.md:25]:
```suggestion
![](img/chat-moderator-profile.png "Page de profil d'un chat moderator en 2013")
```

File: wiki/Client/Options/Keyboard_bindings/fr.md [hunk 8]
```diff
@@ -64,7 +64,7 @@ Ici, vous pouvez modifier la réaffectation des touches du clavier avec les acti
 | Nom | Défaut |
 | :-- | :-- |
 | Toggle Framerate Limit | `F7` |
-| Toggle Chat | `F8` |
+| Toggle tchat | `F8` |
```
Comment (Ganondorf-1) [inline wiki/Client/Options/Keyboard_bindings/fr.md:67]:
```suggestion
| Toggle Chat | `F8` |
```

File: wiki/Gameplay/Spectating/fr.md [hunk 9]
```diff
@@ -10,6 +10,6 @@ tags:
 
 <!-- TODO: needs to be combined with /wiki/Replay#spectator in some way -->
 
-**Spectateur** (abrégé en "spec") est un terme utilisé pour décrire le fait de regarder un autre joueur jouer une [beatmap](/wiki/Beatmap) en temps réel (avec un délai d'environ 500ms). Pour observer, ouvrez le chat étendue et cliquez sur un panneau utilisateur gris.
+**Spectateur** (abrégé en "spec") est un terme utilisé pour décrire le fait de regarder un autre joueur jouer une [beatmap](/wiki/Beatmap) en temps réel (avec un délai d'environ 500ms). Pour observer, ouvrez le tchat étendue et cliquez sur un panneau utilisateur gris.
```
Comment (Ganondorf-1) [inline wiki/Gameplay/Spectating/fr.md:13]:
```suggestion
**Spectateur** (abrégé en "spec") est un terme utilisé pour décrire le fait de regarder un autre joueur jouer une [beatmap](/wiki/Beatmap) en temps réel (avec un délai d'environ 500ms). Pour observer, ouvrez le tchat étendu et cliquez sur un panneau utilisateur gris.
```

File: wiki/Article_styling_criteria/Writing/fr.md [hunk 10]
```diff
@@ -221,7 +221,7 @@ Pour tous les autres termes, osu! doit être traité comme un nom seul. Cela sig
 
 - `osu! tournament`
 - `osu! community`
-- `osu! chat`
+- `osu! tchat`
```
Comment (Biokax) [inline wiki/Article_styling_criteria/Writing/fr.md:224]:
```suggestion
- `osu! tchat`
- `osu! chat`
```

Comment (Biokax) [reply wiki/Article_styling_criteria/Writing/fr.md:224]: garder le tag en anglais comme toujours

**Review by Biokax — CHANGES_REQUESTED** [review]

**Review by Biokax — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**